test/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+[sh_test(
+    name = "{test}".format(test = test),
```

Comment:
While the formatting is neat, wouldn't it be much simpler to just say `name = test` ?

Though probably it would be good to have some prefix or suffix, e.g. a test could be  `tcl_foo` (then of course some sort of string manipulation needed, something like `"tcl_" + test` or `"tcl_{}.format(test)`")